core-server/src/main/java/org/orbisgis/orbisserver/GetCapabilitiesController.java

```
@@ -0,0 +1,111 @@
+ * info_at_ orbisgis.org
+ */
+
+package org.orbisgis.orbisserver;
+
+ import org.orbiswps.server.WpsServer;
+ import org.orbiswps.server.model.JaxbContainer;
+ import org.wisdom.api.DefaultController;
+ import org.wisdom.api.annotations.Controller;
+ import org.wisdom.api.annotations.Parameter;
+ import org.wisdom.api.annotations.Route;
+ import org.wisdom.api.annotations.View;
+ import org.wisdom.api.http.HttpMethod;
+ import org.wisdom.api.http.Result;
+ import org.wisdom.api.templates.Template;
+
+ import javax.xml.bind.JAXBElement;
+ import javax.xml.bind.JAXBException;
+ import javax.xml.bind.Marshaller;
+ import javax.xml.bind.Unmarshaller;
+
+ /**
+  * Instance of DefaultController containing a WpsServer.
+  *
+  * @author Guillaume MANDE
+  */
+ @Controller
+ public class GetCapabilitiesController extends DefaultController {
+
+   /**
+    * IndexController's object, used to get some useful attributes.
+    */
+   private IndexController indexController = new IndexController();
+
+   /**
+    * The action method returning the xml file corresponding to the GetCapabilities method. It handles
+    * HTTP GET request on the "/orbisserv/ows" URL.
+    *
+    * @Parameter the service
+    * @Parameter the version
+    * @Parameter the request GetCpabilities
+    *
+    * @return the xml file
+    */
+   @Route(method = HttpMethod.GET, uri = "/orbisserver/ows")
+   public Result test(@Parameter("service") String service, @Parameter("version") String version, @Parameter("request") String request) {
+       //Simple example of getting information from the WpsServer
+       if(service != null && !service.isEmpty()){
+         if(service.equals("WPS")){
+           if(version != null && !version.isEmpty()){
+             if(version.equals("2.0.0")){
+               if(request!= null && !request.isEmpty()){
+                 if(request.equals("GetCapabilities")){
+                   try { indexController.simpleWpsRequest();  } catch (JAXBException ignored) {}
```

Please avoid to ignore an exception. At least log en error message.